dashboard/app/helpers/aichat_safety_helper.rb

```diff
@@ -34,18 +36,28 @@ def find_toxicity(role, text, locale)
 
     private def openai_safety_check(text)
       details = nil
+      start_time = Time.now
+      report_openai_safety_check("#{self.class.name}.OpenaiStart")
```

Comment:
Instead of using `self.class.name`, could we just use `AichatSafety` or similar? I think the class name shows up as `AichatSafetyHelper::ToxicityDetector` which is pretty long

Could also prepend the `AichatSafety` prefix in the helper function, and just pass in the specific metric name here (`OpenaiStart`)

Also, minor/optional, but we could namespace these with periods for a little more clarity? like `AichatSafety.Openai.Start`, `AichatSafety.Openai.InvalidResponse` - and then we just need to pass in the last part of the metric name (`Start`, `InvalidResponse`, `Finish`)

Reply:
Nice! Will do